src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -436,16 +442,26 @@ private void testCancelEditQuestion() {
         feedbackEditPage.clickDiscardChangesLink(qnIndex);
         feedbackEditPage.waitForConfirmationModalAndClickOk();
         assertFalse(feedbackEditPage.isDiscardChangesButtonVisible(qnIndex));
+        assertFalse(feedbackEditPage.isQuestionEnabled(qnIndex));
         String qnTextAfterCancelEdit = feedbackEditPage.getQuestionBoxText(qnIndex);
         assertEquals(qnTextOriginal, qnTextAfterCancelEdit);
 
+        ______TS("Try cancelling changes to question number");
+        feedbackEditPage.clickEditQuestionButton(qnIndex + 1);
```

Comment:
I believe you should increment `qnIndex` first, since the reader may think `qnIndex` refers to the question you are editing/testing.